### ISSUE TITLE
Database Design

### ISSUE THREAD

**Issue by thackson** [chloedeguzman/Kollab#1]
- [x] 1. Purpose
- [x] 2. Organize Tables 
- [x] 3. Create Relationships and ER Models
- [x] 4. Refine and normalize scheme 
- [x] 5. Implementation

**Comment by thackson** [comment]
So I attached a proposed json file containing the different objects that our database would be working with. 

The json file is made up of 7 objects: User, Bio, Dashboard, Video, Recommendation, Tags, Categories. These objects are made of attributes in the form of strings(or integers), arrays, and documents. (Documents are considered anything that is inside of curly brackets) 

This will not be a json file that we would be working with, but rather a reference to the values that the objects will hold in our database. Since our database and objects will be dynamic, we can include more attributes or leave out some of the attributes that I have provided. 

Let me know if this looks good or not, so that I can get started with the ER-Models and documentation. Also, I had to convert the json file to txt because github doesn't let you use .json files.
[schemas.txt](https://github.com/chloedeguzman/Kollab/files/1334299/schemas.txt)
